server/index.ts

```diff
@@ -17,6 +17,25 @@ import { schema, TypeOf } from '@osd/config-schema';
 import { PluginInitializerContext, PluginConfigDescriptor } from '../../../src/core/server';
 import { SecurityPlugin } from './plugin';
 
+const validateAuthType = (value: string[]) => {
+  const supportedAuthTypes = [
+    '',
+    'basicauth',
+    'jwt',
+    'openid',
+    'saml',
+    'proxy',
+    'kerberos',
+    'proxycache',
+  ];
+
+  value.forEach((authVal) => {
+    if (!supportedAuthTypes.includes(authVal.toLowerCase())) {
+      return `allowed auth.type are ${supportedAuthTypes}`;
```

Comment:
I would add a clear message "Some of the configured auth types are not supported" or something like this.